mlflow/gateway/client.py

```diff
@@ -337,3 +338,77 @@ def query(self, route: str, data: Dict[str, Any]):
                 raise MlflowException(message=timeout_message, error_code=BAD_REQUEST)
             else:
                 raise e
+
+    @experimental
+    def set_limits(self, route_name: str, limits_config: LimitsConfig) -> LimitsConfig:
```

Comment:
```suggestion
    def set_limits(self, route: str, limits_config: LimitsConfig) -> LimitsConfig:
```

Can we use `route` to be consistent with `query`?

Reply:
sg!